sharunkumar/AdventOfCode
# Day 6: Wait For It

- initialize files
- populate input files
- temp comment part 2 test
- update test expect
- tests passing for part 1
- refactor using functions
- test against prod input
- prepare part 2 tests
- copy over part 1 code
- tests passing for part 2
- works with prod input
